api/src/org/labkey/api/files/view/FilesWebPart.java

```diff
@@ -409,6 +410,8 @@ private static String _getRootPath(Container c, @Nullable String relativePath, b
         if (!rootPath.endsWith("/"))
             rootPath += "/";
 
+        // Issue 43374: Some characters in file names cause a problem when creating exp.data rows in FileImporter.
+        // This fix requires decoding these characters so that they don't get encoded twice - see getURLWithCharsReplaced() in Webdav.js and its usages.

```

Comment:
I find this comment confusing, since the line it's attached to is encoding, not decoding.
```suggestion
        // Need to centrally encode characters that are special in URIs but not in most file systems. See getURLWithCharsReplaced() in Webdav.js for special client-side handling of these characters.

```